pmm/conan.cmake

```diff
@@ -710,6 +710,22 @@ function(_pmm_conan_ensure_remotes remotes)
     endwhile()
 endfunction()
 
+function(_pmm_generate_conan_cli_scripts)
+    # The sh scipt
+    if (NOT EXISTS "${CMAKE_BINARY_DIR}/conan.sh")
+        file(WRITE "${_PMM_USER_DATA_DIR}/conan.sh" "#!/bin/sh\n\"${PMM_CONAN_EXECUTABLE}\" \"$@\"")
+        # Fix to make the sh executable
+        file(COPY "${_PMM_USER_DATA_DIR}/conan.sh"
+                DESTINATION ${CMAKE_BINARY_DIR}
+                FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+                )
+    endif ()
```

Comment:
This "write script, then copy to make executable" appears a few times, and might be useful enough to make a util function to do it.